dada2_inference.R

```diff
@@ -0,0 +1,299 @@
+#!/usr/bin/env Rscript
+
+# Read in package to read in command-line options.
+library("optparse")
+
+version <- "1.0"
+
+option_list <- list(
+
+  make_option(c("-f", "--f_path"), type="character", default=NULL,
+              help="Location of forward reads (required)." , metavar="path"),
+  
+  make_option(c("--seed"), type="integer", default=NULL,
+              help="Random seed to make command reproducible (required).", metavar = "integer"),
+  
+  make_option(c("-r", "--r_path"), type="character", default=NULL,
+              help="Location of reverse reads (if applicable). Same as forward reads by default.",
+              metavar="path"),
+  
+  make_option(c("-s", "--single"), action = "store_true", type="logical", default=FALSE,
+              help="Flag to indicate that reads are single-end (default: FALSE).", metavar = "boolean"),
+
+  make_option(c("--f_match"), type="character", default="_R1_.*fastq.*",
```

Comment:
I like this option, since file naming seems to vary somewhat for different datasets.